autotx/utils/ethereum/load_w3.py

```diff
@@ -1,6 +1,11 @@
+import os
 from web3 import Web3
-from autotx.utils.ethereum.constants import FORK_RPC_URL
+from autotx.utils.ethereum.constants import DEVNET_RPC_URL
+from autotx.utils.is_dev_env import is_dev_env
 
 def load_w3() -> Web3:
-    web3 = Web3(Web3.HTTPProvider(FORK_RPC_URL))
+    rpc_url = DEVNET_RPC_URL if is_dev_env() else os.getenv("CHAIN_RPC_URL")
+    
+    web3 = Web3(Web3.HTTPProvider(rpc_url))
```

Comment:
we do the same in `get_configuration`, with the difference that we throw is `CHAIN_RPC_URL` is not set - should we add that validation here and use `load_w3` in `get_configuration` ?

Reply:
I added the check there, but we cant reuse `load_w3` since we also need the `rpc_url` for `EthereumClient`